a/animal-sniffer/animal-sniffer_ubi_8.5.sh

```diff
@@ -0,0 +1,45 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : animal-sniffer
+# Version       : animal-sniffer-parent-1.17 / animal-sniffer-parent-1.14
+# Source repo   : https://github.com/mojohaus/animal-sniffer
+# Tested on		: UBI 8.5
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Saurabh Gore <Saurabh.Gore@ibm.com> 
+#
+# Disclaimer: This script has been tested in root mode on given
+# ========== platform using the mentioned version of the package.
+# It may not work as expected with newer versions of the
+# package and/or distribution. In such case, please
+# contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+
+#Variables
+PACKAGE_NAME=animal-sniffer
+PACKAGE_VERSION="${1:-animal-sniffer-parent-1.17}"
+PACKAGE_URL=https://github.com/mojohaus/animal-sniffer.git
+
+
+# Install required files
+yum install -y git maven java-1.8.0-openjdk-devel
```

Comment:
Just a suggestion..
`java` & `java-devel` will get installed automatically, If we are installing `maven`. No need of explicit installation.